src/Umbraco.Web/PropertyEditors/TextAreaPropertyEditor.cs

```diff
@@ -20,6 +20,9 @@ internal class TextAreaPreValueEditor : PreValueEditor
         {
             [PreValueField("maxChars", "Maximum allowed characters", "number", Description = "If empty - no character limit")]
             public bool MaxChars { get; set; }
+
+            [PreValueField("rows", "Number of rows", "number", Description = "If empty - 10 rows would be set as the default value")]
+            public bool Rows { get; set; }
         }

```

Comment:
Shouldn't the properties `MaxChars` and `Rows` be of type `int` - instead of type `bool`?

Reply:
@bjarnef that's a really good point, to be honest, I have only added the rows and I was wondering why the other one is set to bool that's why I set the rows of type bool. Not sure what was the reason for that. it make sense for both properties to be of type int.

Reply:
Should I update that to int then? if so how do I pass the description and the field label?

Reply:
Yes, please :) that the correct type. I fixed some similar property types here https://github.com/umbraco/Umbraco-CMS/pull/2326/files but I have no idea why e.g. TabName did seems to work while the property was of type int, but it was a bit misleading.